### PR TITLE
Rider suggested cleanup, part 1 

### PR DESCRIPTION
**Pull request by chriselion**
### Proposed change(s)
Low-hanging fruit, mostly unused `using ...`

### Types of change(s)
- [x] Bug fix

### CODE REVIEW

File: Project/Assets/ML-Agents/Examples/Sorter/Scripts/NumberTile.cs [hunk 1]
```diff
@@ -8,7 +6,7 @@ public class NumberTile : MonoBehaviour
     public Material DefaultMaterial;
     public Material SuccessMaterial;
 
-    private bool m_Visited = false;
```
Comment (chriselion) [inline Project/Assets/ML-Agents/Examples/Sorter/Scripts/NumberTile.cs:11]: Not sure how people feel about. My instinct is to a) be explicit (from python) and b) always initialize everything (coming from C++), but Rider likes to complain about it.

File: com.unity.ml-agents/Runtime/MLAgentsSettingsManager.cs [hunk 2]
```diff
@@ -1,8 +1,9 @@
 using System;
-using System.Linq;
```
Comment (chriselion) [inline com.unity.ml-agents/Runtime/MLAgentsSettingsManager.cs:2]: This is only needed in play mode.

File: Project/Assets/ML-Agents/Examples/Sorter/Scripts/NumberTile.cs (hunk 1 above)
Comment (dongruoping) [reply Project/Assets/ML-Agents/Examples/Sorter/Scripts/NumberTile.cs:11]: Yeah I think making them explicit is not a bad thing and sometimes it feels safer (I know it works that way but I like seeing it written that way).
No strong opinions anyway.

**Review by dongruoping — APPROVED** [review]

File: Project/Assets/ML-Agents/Examples/Sorter/Scripts/NumberTile.cs (hunk 1 above)
Comment (surfnerd) [reply Project/Assets/ML-Agents/Examples/Sorter/Scripts/NumberTile.cs:11]: _traumatized by c++_

**Review by surfnerd — COMMENTED** [review]
LGTM

**Pull request merged**